assignment-2-ruby/chimi/Question 2.rb

```diff
@@ -0,0 +1,143 @@
+# Question 2. Tiny Instacart!
+# Compose a simple terminal program using one or more of the common objects from Instacart (orders, drivers, batches, order_items, payments, etc)
+# Design a desired workflow. ex. The entire flow of a driver from accepting a batch to delivery
+# Using the tools and concepts from the completed room cleaner program, construct a self-running terminal program that successfully runs through your intended workflow.
+# Test your program thoroughly to ensure that it runs as expected.
+
+# Possible Extensions:
+# Ruby offers alternatives to if/else including unless, while, and until.
+# Set parameters for your program through user input
+# Make sure to incorporate if/else and the appropriate operators.
+# Look up how to use a ruby hash to create an array with multiple attributes per array element and run an each loop that checks the values of said attributes. 
+# ex. (`drivers = [{name: "Rose", role: "in-store-shopper"}, {name: "Donna", role: "shift-lead"}, {name: "Amelia Pond", role: "full-service"}, {name: "Martha Jones", role: "cashier"})
+
+def customer_name
+    customer_name = "A.B"
+end
+
+def customer_address
+    customer_address = "50 Beale St, San Francisco, CA 94105."
+end
+
+def store_information
+    store_information = "H-E=B. 11815 Westheimer Rd, Houston"
+end
+
+def grocery_list
+    grocery_list = [{item: "Banana", quantity: 5}, {item: "Quaker's Oats", quantity: 2}, {item: "Fresh Squeezed Orage Juice", quantity: 1}]
+end
+
+def assistance
+    puts "Please call CARE for assistance."
+end
+
+def batch_earnings
+    puts "This is a full service batch with delivery to 1 customer."
+    puts "Total Earnings: $23.57"
```

Comment:
Let's update that to take into consideration the actual `grocery_list` (How many items should change the earnings)